src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -633,7 +631,7 @@ def __call__(self):
             if funcName == "repack":
                 self.handleRepackSettings()
 
-            if funcName in ["repack", "merge", "alcaHarvesting" ]:
+            if funcName in ["merge", "alcaHarvesting" ]:
```

Comment:
Can you remind me why they are multicore for HTCondor but must run single core? Is it about job requirements that are put in the jdl? If so, then we should try to make these changes at spec level and make sure the job/performance requirements are set since the beginning and avoid hacking around during job submission or runtime.